src/pages/workspaces/workspace/Data.js

```diff
@@ -729,18 +729,18 @@ export const WorkspaceData = _.flow(
     };
 
     const loadWdsUrl = useCallback((workspaceId) => {
-      // setWdsProxyUrl({ status: 'Loading', state: '' })
       return Ajax()
         .Apps.listAppsV2(workspaceId)
-        .then(resolveWdsUrl)
-        .then((url) => {
-          if (url) {
+        .then((apps) => {
+          const foundApp = resolveWdsApp(apps);
+          if (foundApp?.status === 'RUNNING') {
+            const url = foundApp.proxyUrls.wds;
```

Comment:
Do we need this to be `.proxyUrls?` or is that unnecessary caution?

Reply:
According to the type definition for `ListAppResponse`, `proxyUrls` will be defined.

Reply:
Aha!  Thank you typescript!